be/src/storage/lake/lake_local_persistent_index.cpp

```diff
@@ -64,10 +64,11 @@ Status LakeLocalPersistentIndex::load_from_lake_tablet(starrocks::lake::Tablet*
         // here just use version.major_number so that PersistentIndexMetaPB need't to be modified,
         // the minor_number is meaningless
         if (version.major_number() == base_version) {
-            // If format version is not equal to PERSISTENT_INDEX_VERSION_2, this maybe upgrade from
+            // If format version is not equal to PERSISTENT_INDEX_VERSION_3, this maybe upgrade from
             // PERSISTENT_INDEX_VERSION_2.
             // We need to rebuild persistent index because the meta structure is changed
-            if (index_meta.format_version() != PERSISTENT_INDEX_VERSION_2) {
+            if (index_meta.format_version() != PERSISTENT_INDEX_VERSION_2 &&
+                index_meta.format_version() != PERSISTENT_INDEX_VERSION_3) {
```

Comment:
Is PERSISTENT_INDEX_VERSION_3 backwards compatible with PERSISTENT_INDEX_VERSION_2？

Reply:
Yes